Tutorials/Day_1/Tuto_1.2_Open_Data_access_with_GWpy.ipynb

```diff
@@ -365,10 +365,7 @@
     "id": "1irX8-UnvvXd"
    },
    "source": [
-    "Notes: There are alternative ways to access the GWOSC data. \n",
-    "\n",
-    "* [`readligo`](https://losc.ligo.org/s/sample_code/readligo.py) is a light-weight Python module that returns the time series into a Numpy array.\n",
-    "* The [PyCBC](http://github.com/ligo-cbc/pycbc) package has the `pycbc.frame.query_and_read_frame` and `pycbc.frame.read_frame` methods. We use [PyCBC](http://github.com/ligo-cbc/pycbc) in Tutorial 2.1, 2.2 and 2.3. "
+    "Note: There are alternative ways to access GWOSC data. The [PyCBC](http://github.com/ligo-cbc/pycbc) package has the `pycbc.frame.query_and_read_frame` and `pycbc.frame.read_frame` methods that we will learn in tutorials 2.1-3."
```

Comment:
I have a non-blocking remark: it took me a few seconds to understand what you mean by "tutorials 2.1-3"; I am not sure we often refer to tutorials like that (although the previous sentence used "Tutorials 2.1") so maybe replace that by "Tutorials 2.1 to 2.3".

Reply:
Ok, I changed it to your suggestion